projects/packages/sync/src/modules/class-full-sync-immediately.php

```diff
@@ -80,7 +80,14 @@ public function start( $full_sync_config = null ) {
 		$this->reset_data();
 
 		if ( ! is_array( $full_sync_config ) ) {
-			$full_sync_config = Defaults::$default_full_sync_config;
+			/*
+			 * Filer defualt sync config to allow injecting custom configuration.
+			 *
+			 * @param array $full_sync_config Sync configuration for all sync modules.
+			 *
+			 * @since x.x.x
```

Comment:
```suggestion
			 * @since $$next-version$$
```

The `$$next-version$$` specifier will be [automatically replaced with the correct package version number](https://github.com/Automattic/jetpack/blob/ff2476c16e5ca08852f02601db8660988ff13c94/projects/packages/README.md?plain=1#L116) the next time a new version of that package is released.